lib/screens/request_board_approved_screen.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:cobe_hive_mobile_app/providers/leave_request_provider.dart';
+import 'package:cobe_hive_mobile_app/widgets/expandable_fab.dart';
+import 'package:cobe_hive_mobile_app/widgets/request_board_header.dart';
+import 'package:cobe_hive_mobile_app/widgets/approvedd_request_list.dart';
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class RequestBoardApprovedScreen extends ConsumerWidget {
+  const RequestBoardApprovedScreen({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      body: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 18),
+        child: Column(
+          children: [
+            const Padding(
+              padding: EdgeInsets.only(bottom: 40, top: 18),
+              child: RequestBoardHeader(),
+            ),
+            const Row(
+              children: [
+                Text(
+                  'Approved',
+                  style: TextStyle(
+                    fontSize: 24,
+                    fontWeight: FontWeight.w500,
+                  ),
+                ),
+                Expanded(
+                    child: SizedBox(
+                  height: 30,
+                )),
+              ],
+            ),
+            Expanded(
+              child: ApprovedRequestList(
+                leaveRequestList: ref.watch(leaveRequestListApprovedProvider),
```

Comment:
This way the whole  `RequestBoardApprovedScreen ` rebuilds when an approved `LeaveRequest` is added. Only the list should rebuild. Move `ref.watch(leaveRequestListApprovedProvider)` into `ApprovedRequestList `